### PR TITLE
docs: add events page to track our events

### LINKED ISSUE #875: Add a online meetup page
**Actual Behaviour**

Currently, Apache APISIX don't have meetup page, if someone want to participate online meetup, they only receive invitation from mailling list.

 

**Expected Behaviour**

A better way is let users subscribe meetup link, just like https://konghq.com/online-meetups/

 

**Steps to reproduce it**

 

**Screenshots of the issue**

 

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by Boburmirzo**
Fixes: #875

Changes:

I added a new Events page to provide information about upcoming Apache APISIX community events where you can track Meetups, Conferences, and Workshops.

Here I entered only a single conference to show you the structure (How it looks like). If we are okay with the current template, I will fill it with other events data. 

I have a plan also to add a new GitHub issue template for adding new events so that anyone can create a PR and add their scheduled events.

Screenshots of the change:

bFeBzff3Uw

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by juzhiyuan — DISMISSED** [review]
LGTM

**Comment by juzhiyuan** [comment]
Hi, @Boburmirzo, would you please check this CI? https://github.com/apache/apisix-website/runs/5676296051?check_suite_focus=true 😄

**Comment by Boburmirzo** [comment]
> Hi, @Boburmirzo, would you please check this CI? https://github.com/apache/apisix-website/runs/5676296051?check_suite_focus=true 😄

I will fix it.

**Comment by Boburmirzo** [comment]
> Hi, @Boburmirzo, would you please check this CI? https://github.com/apache/apisix-website/runs/5676296051?check_suite_focus=true 😄

Now it is fixed.

**Review by takumi913 — DISMISSED** [review]

File: website/config/navbar.js [hunk 1]
```diff
@@ -78,6 +78,10 @@ module.exports = [
         to: "/docs/general/community",
         label: "Community"
       },
+      {
+        to: "/docs/general/events",
+        label: "Events"
+      },
```
Comment (guoqqqi) [inline website/config/navbar.js:84]: Is it necessary to put Events in the navigation, where each item is usually on a different page? 🤔

Comment (juzhiyuan) [reply website/config/navbar.js:84]: > each item is usually on a different page? 

Hi, could you share more details on this? I'm not clear what do you mean 😅

File: website/docs/general/events.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: Events
+keywords:
+  - Events
+  - Conferences
+  - Workshops
+  - Meetups
+description: This page provides information about upcoming Apache APISIX's community events where you can track Meetups, Conferences and Workshops.
+---
+
+## Events
```
Comment (juzhiyuan) [inline website/docs/general/events.md]: Hi, bcoz we have the Page Title attribute, how about removing this one?

**Review by juzhiyuan — DISMISSED** [review]
LGTM

File: website/config/navbar.js (hunk 1 above)
Comment (guoqqqi) [reply website/config/navbar.js:84]: image
Sorry, I meant that Community and Events are different categories on the same page. So, do we need to add events to the navigation? 😯

Comment (juzhiyuan) [reply website/config/navbar.js:84]: After a quick confirmation with @guoqqqi, Qi means why we need to put the `Events` and `Community` menu in the same category dropdown, 😅 bcoz when users visit Community, they will also notice Events. 

We think it's ok to keep the current state.

File: website/docs/general/events.md (hunk 2 above)
Comment (Boburmirzo) [reply website/docs/general/events.md]: Good point! Let's remove that.

Comment (Boburmirzo) [reply website/docs/general/events.md]: > Hi, bcoz we have the Page Title attribute, how about removing this one?

I removed the title of the event but added content navigation for different kind of events like:

## Contents

* [Conferences](#conferences)
* [Workshops](#workshops)
* [Meetups](#meetups)

image


But again we need some approvals from zero:)

Comment (Boburmirzo) [reply website/docs/general/events.md]: @juzhiyuan Now it looks better.

Comment (juzhiyuan) [reply website/docs/general/events.md]: Hi, Bobur, actually the right sidebar contains ToC 😂

Comment (Boburmirzo) [reply website/docs/general/events.md]: > Hi, Bobur, actually the right sidebar contains ToC 😂

Yes, now can we proceed with current solution?

Comment (Boburmirzo) [reply website/docs/general/events.md]: image

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by bzp2010 — APPROVED** [review]

**Review by SkyeYoung — COMMENTED** [review]
LGTM

**Comment by Boburmirzo** [comment]
Thanks All!

**Pull request merged**